cloudinit/util.py

```diff
@@ -520,13 +520,25 @@ def multi_log(text, console=True, stderr=True,
 
 @lru_cache()
 def is_BSD():
-    return 'BSD' in platform.system()
+    if 'BSD' in platform.system():
+        return True
+    if platform.system() == 'DragonFly':
```

Comment:
It starts with Capital letter unlikely others. Is it specific form this way?

Reply:
There is no trailing 'BSD':

```
[root@basehost /home/goneri/builder]# python3
Python 3.7.6 (default, Jan 30 2020, 11:44:54) 
[GCC 8.3 [DragonFly] Release/2019-02-22] on dragonfly5
Type "help", "copyright", "credits" or "license" for more information.
>>> import platform
>>> platform.system()
'DragonFly'
```